static/js/settings_user_groups.js

```diff
@@ -255,6 +208,60 @@ exports.populate_user_groups = function () {
     });
 };
 
+exports.set_raw_description = function (target, destination) {
+    var group_id = $(target).closest('.user-group').attr("id");
+    var group_data = user_groups.get_user_group_from_id(group_id);
+    if (!group_data) {
+        blueslip.error('set_raw_description() fails');
+        return;
+    }
+    destination.text(group_data.description);
+};
+
+function change_name_desc(e, flag) {
+    e.preventDefault();
+    var target = [" .name", " .description"];
+    var group_id = $(e.target).closest(".user-group").attr("id");
+    var user_group_status = $('#user-groups #' + group_id + ' .user-group-status');
+    var group_data = user_groups.get_user_group_from_id(group_id);
+    var changed_val = $('#user-groups #' + group_id + target[flag]).text().trim();
```

Comment:
Also here, instead of passing flag, I would suggest to pass `name` or `description` argument. I understand that list would be more precise here, but that makes the function complex. Instead you can add `if` condition to check if element exist if not throw an error.

Reply:
should I pass ` .name` or ` .description` as arguments ?